src/lib/xmlparse.rs

```diff
@@ -1677,13 +1669,14 @@ impl XML_ParserStruct {
         }
         if !nameSep.is_null() {
             parser.m_ns = XML_TRUE;
-            parser.m_namespaceSeparator = *nameSep
+            parser.m_namespaceSeparator = *nameSep;
         }
 
         #[cfg(feature = "mozilla")]
```

Comment:
I think this is redundant now, you can remove it.

Reply:
Sure, but it's good style if there isn't a return value from the block.

Reply:
I meant the assignment to `m_mismatch`, not the semicolon (that one's fine).

Reply:
Oh, dunno, it was already there.

Reply:
Yeah I added it, but now that building the `XML_ParserStruct` and initializing it happen in the same function, there is not point for it anymore (not that there was much to begin with, but that's what Mozilla's original C code did).